src/config/validate.ts

```diff
@@ -101,6 +102,13 @@ function validateUrlLifetime(urlLifetime: string): void {
 	}
 }
 
+function validateUrlExpireFrom(urlExpire: string): void {
+	logger.debug(`Start validateUrlExpireFrom with ${urlExpire}`)
+	if (!urlExpire || (urlExpire != 'create' && urlExpire != 'update')) {
```

Comment:
Don't use `!=` or `==` in JS. Use `!==` and `===` instead.
```suggestion
	if (!urlExpire || (urlExpire !== 'create' && urlExpire !== 'update')) {
```